react-app/src/components/Result.js

```diff
@@ -36,6 +36,45 @@ function Result(props) {
     return total;
   }
 
+  function getLabel() {
+    // attempt to get MRN and name with fhirpath
+    const patient = fhirpath.evaluate(props.bundle, "Bundle.descendants().resource.where(resourceType='Patient')")[0];
+    if (patient) {
+      const mrn = patient.id;
+      const name = patient.name[0].text;
+
+      let isMasked = fhirpath.evaluate(patient, 'Patient.identifier.extension.valueCode')[0];
+
+      if (isMasked === 'masked') {
+        isMasked = true;
+      } else {
+        isMasked = false;
+      }
```

Comment:
Surely we could define this with a `const` without the conditional here, right?

Reply:
I have modified this so that it's a const and only one line.